src/types/api/sequencer.ts

```diff
@@ -259,6 +264,13 @@ export namespace Sequencer {
       REQUEST: never;
       RESPONSE: TransactionReceiptResponse;
     };
+    get_nonce: {
+      QUERY: {
+        contractAddress: string;
+      };
+      REQUEST: never;
+      RESPONSE: any;
```

Comment:
is it now BigNumberish here?
https://github.com/0xs34n/starknet.js/pull/318/files#diff-c6400b98d0fbb382983009129ced9699b8471de4e5fda478add242afa995ab4fR232